Sources/Misc/StoreKitVersion.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreKitVersion.swift
+//
+//  Created by Mark Villacampa on 4/13/23.
+
+import Foundation
+
+/// Defines which version of StoreKit may be used
+@objc(RCStoreKitVersion)
+public enum StoreKitVersion: Int {
```

Comment:
Do you need this to have a `rawValue`? I'd remove it unless we're using it. And if we are, I'd explicitly set the values (right now `storeKit1` is `0`, and `storeKit2` is `1`)
```suggestion
public enum StoreKitVersion {
```

Reply:
`@objc' enum must declare an integer raw type`